feat: add `{line,fill}-layer-opacity` (#1650)
maplibre/maplibre-style-spec: CHANGELOG.md

## main
### ✨ Features and improvements

> - Added `layer-opacity` paint property to `fill` and `line` layers - applies opacity to the whole layer as a single composite so overlapping features stay uniform. ([#1650](https://github.com/maplibre/maplibre-style-spec/pull/1650)) (by [@CommanderStorm](https://github.com/CommanderStorm))

Author: ```suggestion
- Added `{line,fill}-layer-opacity` paint property. It applies opacity to the whole layer as a whole layer so overlapping features change in an uniform way. ([#1650](https://github.com/maplibre/maplibre-style-spec/pull/1650)) (by [@CommanderStorm](https://github.com/CommanderStorm))
```
